vuepress/docs/next/docs/concepts/operator-intro.md

```diff
@@ -0,0 +1,64 @@
+# The Entando Operator
+
+The Entando Operator processes the custom resources in Kubernetes that represent the different [components of an Entando application](./README.md).
+The goal of the operator is to provide automation and a set of default configuration to simplify and accelerate the deployment of an Entando application.
+
+The sections below provide details and assumptions that the operator makes when deploying Entando Custom Resources. If you're using OpenShift these
+sections will provide details on how to configure your deployment via the Operator Hub.
+
+For details on the individual custom resources and their configuration check out the [custom resources documentation](./custom-resources.md).
+
+## TLS Secret Creation
+
+When configuring and deploying Entando via the operator you will be asked to provide a secret for some of the components in the architecture. A few things to be aware of when creating and configuring a secret:
+
+- The secret is assumed to be in the same namespace as the component being created.
+- This Secret is expected to have a private key, and a certificate for the hostname that the service
```

Comment:
lowercase Secret to be consistent?

Reply:
Per this line, it could also be a wildcard cert.

Reply:
done